zone/special_attacks.cpp

```diff
@@ -182,7 +183,7 @@ int Mob::GetBaseSkillDamage(EQ::skills::SkillType skill, Mob *target)
 				}
 			} else if (IsNPC()) {
 				auto *npc = CastToNPC();
-				base = std::max(base, npc->GetBaseDamage());
+				base = round((npc->GetMaxDMG() + npc->GetMinDMG()) / RuleR(NPC, NPCBackstabMod));
```

Comment:
Where does this formula come from

Reply:
https://github.com/EQEmu/Server/blob/master/zone/npc.cpp#L216